src/Compiler/Build/WebForms.Compiler.DefaultItems.props

```diff
@@ -11,17 +11,19 @@
   <ItemGroup>
     <WebFormsFiles Include="$(MSBuildProjectDirectory)\**\*.aspx" Exclude="@(__DefaultExcludes)" />
     <WebFormsFiles Include="$(MSBuildProjectDirectory)\**\*.master" Exclude="@(__DefaultExcludes) "/>
+    <WebFormsFiles Include="$(MSBuildProjectDirectory)\**\*.ascx*" Exclude="@(__DefaultExcludes)" />
 
     <WebFormsDesignerFiles Include="$(MSBuildProjectDirectory)\**\*.aspx.designer.*" Exclude="@(__DefaultExcludes)" />
     <WebFormsDesignerFiles Include="$(MSBuildProjectDirectory)\**\*.master.designer.*" Exclude="@(__DefaultExcludes) "/>
```

Comment:
add a `ascx.designer.*`

Reply:
note - these changes should wait until we actually support user controls. if you do break things into incremental prs, then this should be the final one.